.travis.yml

```diff
@@ -8,4 +8,4 @@ before_script:
   - curl -s http://getcomposer.org/installer | php
   - php composer.phar install
 
-script: phpunit test/Tests
+script: ./vendor/bin/phpunit
```

Comment:
If we add `"scripts": {"test": "phpunit"}` to `composer.json`, then `.travis.yml`'s `script:` could become `composer test`.